AdyenComponents/Apple Pay/ApplePayComponent.swift

```diff
@@ -83,94 +57,62 @@ public class ApplePayComponent: NSObject, PaymentComponent, PresentableComponent
         guard configuration.summaryItems.filter({ $0.amount.isEqual(to: NSDecimalNumber.notANumber) }).count == 0 else {
             throw Error.invalidSummaryItem
         }
-        
+        let request: PKPaymentRequest = configuration.createPaymentRequest()
+        guard let paymentAuthorizationViewController = ApplePayComponent.createPaymentAuthorizationViewController(from: request) else {
+            throw UnknownError(
+                errorDescription: "Failed to instantiate PKPaymentAuthorizationViewController because of unknown error"
+            )
+        }
+
         self.configuration = configuration
+        self.paymentAuthorizationViewController = paymentAuthorizationViewController
```

Comment:
`self.paymentAuthorizationViewController` doesn't have to be optional, right?

Reply:
It has to be because PKPaymentAuthorizationViewController can't be presented twice. We have to kill and recreate it every time

Reply:
👍 yeah, forgot about that